doc/compiler/jitserver/Usage.md

```diff
@@ -20,143 +20,243 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# Usage
+# Getting started with JITServer
 
-This document describes how to use JITServer technology with any Java application and some basic options, such as logging and encryption.
+This document explains how to use JITServer technology with any Java application and describes some basic options, such as network configuration, logging and encryption.
+
+1. [Running applications with JITServer](#running-applications-with-jitserver)
+2. [Network Configuration](#network-configuration)
+3. [JITServer-specific options](#jitserver-specific-options)
+4. [Logging](#logging)
 
 ## Running applications with JITServer
 
-JITServer adds two additional *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer` (under bin run `./java -XX:+UseJITServer <application name>`)
+JITServer adds two additional *personas* to the JVM: **client** mode and **server** mode.
+**Client** is the Java application you want to run, and **server** performs JIT compilations for the **client**.
+
+In **server** mode, the JVM will halt after startup and begin listening for compilation requests from clients. To start the **server**, run the `jitserver` launcher present under `bin` directory (alongside `java` launcher). No Java application is given to the `jitserver` launcher on the command line. Any option accepted by JVM can also be passed to the `jitserver` launcher. One **server** is capable of serving multiple clients.
 
-In server mode, the JVM will halt after startup and begin listening for compilation requests from clients. No Java application is given to the `jitserver` launcher on the command line. Any option accepted by JVM can also be passed to the `jitserver` launcher.
+Begin by launching the **server**:
 
 ```
-$ jitserver
+$ jitserver -XX:+JITServerLogConnections
```

Comment:
I thought it would be a nice option to use for this tutorial, since it would allow users to see that client and server are actually talking to each other.

Reply:
ok